dev-games/tiled/tiled-1.11.0.ebuild

```diff
@@ -0,0 +1,65 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit xdg-utils
+
+DESCRIPTION="A general purpose tile map editor"
+HOMEPAGE="https://www.mapeditor.org/"
+SRC_URI="https://github.com/mapeditor/tiled/archive/v${PV}/${P}.tar.gz"
+
+LICENSE="BSD BSD-2 GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="clang examples"
+
+RDEPEND="
+	app-arch/zstd
+	dev-qt/qtbase:6[X]
+	dev-qt/qtimageformats:6
+	dev-qt/qtdeclarative:6
+	dev-qt/qtwayland:6
+	sys-libs/zlib
+"
+DEPEND="${RDEPEND}"
+BDEPEND="
+	virtual/pkgconfig
+	dev-util/qbs
+"
+
+src_configure() {
+	qbs setup-qt /usr/bin/qmake6 qt6 || die
+	qbs config defaultProfile qt6 || die
+
+	if use clang; then
+		qbs setup-toolchains clang clang || die
+		qbs config profiles.qt6.baseProfile clang || die
+	else
+		qbs setup-toolchains gcc gcc || die
+		qbs config profiles.qt6.baseProfile gcc || die
+	fi
+
+	qbs qbs.installPrefix:"/usr" projects.Tiled.installHeaders:true project.libDir:$(get_libdir) || die
```

Comment:
Could move building to src_compile and also include the use of the multiprocessing eclass to respect user MAKEOPTS so that qbs doesnt use max nproc.

Reply:
I think it would be wiser to make eclass for building with QBS than handling all the compiler flags and MAKEOPTS in this ebuild.

Reply:
Its best to first figure them out in an ebuild. Eclasses are good when you have more ebuilds that need the same handling, starting off with an eclass is more restricting as it needs mailing list review and such.